### PR TITLE
Use --enable=automation-service-broker for v3.10

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
We are danger close: https://github.com/openshift/origin/pull/19409

**Comment by jmontleon** [comment]
- This PR also adds back --image for v3.10, which is now fixed.
- It also adds sync'ing the openshift-control-plane image for v3.10

**Comment by jmontleon** [comment]
You'lll probably notice oc cluster up takes longer to run, but it should complete. It's waiting for the apb job to complete, which takes about two minutes so that's on us.

File: ansible/roles/openshift_setup/tasks/main.yml [hunk 1]
```diff
@@ -256,7 +256,7 @@
         --base-dir={{ oc_cluster_up_base_dir }}
         --tag={{ origin_image_tag }}
         --image={{ origin_image_name }}-\${component}:\${version}
-        --enable=service-catalog,template-service-broker,router,registry,web-console,persistent-volumes,sample-templates,rhel-imagestreams
+        --enable=service-catalog,template-service-broker,router,registry,web-console,persistent-volumes,sample-templates,rhel-imagestreams,automation-service-broker
```
Comment (djzager) [inline ansible/roles/openshift_setup/tasks/main.yml]: do we skip the `ansible_service_broker_setup` for 3.10?

Comment (djzager) [reply ansible/roles/openshift_setup/tasks/main.yml]: :sunglasses: Sometimes I don't read so good.

**Comment by jwmatthews** [comment]
Tested on Mac and had success with provision/bind of mediawiki/postgres.

ACK

**Pull request merged**